driver/run.go

```diff
@@ -10,13 +10,21 @@ import (
 )
 
 type Driver struct {
-	writer   zbuf.Writer
+	writers  []zbuf.Writer
 	warnings io.Writer
+	demux    bool
 }
 
 func New(w zbuf.Writer) *Driver {
 	return &Driver{
-		writer: w,
+		writers: []zbuf.Writer{w},
+	}
+}
+
+func NewDemuxed(ws []zbuf.Writer) *Driver {
+	return &Driver{
+		writers: ws,
+		demux:   true,
 	}
 }
```

Comment:
Could we simplify this to
```go
type Driver struct {
	writers  []zbuf.Writer
	warnings io.Writer
}

func New(w ...zbuf.Writer) *Driver {
	return &Driver{
		writers: w,
	}
}
```
and use `len(d.writers) > 1` in place of `d.demux`?

Reply:
@nwt much better. Thanks!